app/src/main/kotlin/io/intrepid/skotlinton/base/BaseScreen.kt

```diff
@@ -0,0 +1,3 @@
+package io.intrepid.skotlinton.base
+
+interface BaseScreen
```

Comment:
Why Screen instead of View?

Reply:
So when we have a wrapper interface (BaseContract) `View` is a nice definition of the indirection we're creating. Since this PR addresses the fact that we don't really need a Presenter interface (just public methods in our BasePresenter), I've changed the name to screen which seems to be idiomatic and informative.